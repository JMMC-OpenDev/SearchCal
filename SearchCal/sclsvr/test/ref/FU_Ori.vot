version https://git-lfs.github.com/spec/v1
oid sha256:f369c604ad80de45ba8c89bc03310436aaa17a98133d5c0f64d950a6e6642a15
size 73354
