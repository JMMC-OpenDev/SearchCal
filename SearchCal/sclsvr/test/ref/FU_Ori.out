version https://git-lfs.github.com/spec/v1
oid sha256:4d834f7e0f9f32af5501d9d7d3ff0922fb48ae541643eb154c5f4ca3c78b29fd
size 321214
